src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+function sortList(roster) {
+  const rosterItems = [...roster.querySelectorAll('li')];
```

Comment:
it can be
```suggestion
 const rosterItems = [...roster.querySelectorAll('li')];
 
function sortList(roster) {
```
looks like it can be find all li one time and use variable in both functions